front-end/src/app/team/team.component.ts

```diff
@@ -1,10 +1,62 @@
-import { Component } from '@angular/core';
+import { Component, ViewChild } from '@angular/core';
+import { AgGridAngular } from 'ag-grid-angular';
+import { ColDef, GridReadyEvent } from 'ag-grid-community';
+import { Observable } from 'rxjs';
+import { Player } from '../types/Player';
+import { Team } from '../types/Team';
+import { ActivatedRoute } from '@angular/router';
+import { TeamService } from '../team-service/team.service';
+import { Club } from '../types/Club';
 
 @Component({
   selector: 'app-team',
   templateUrl: './team.component.html',
-  styleUrls: ['./team.component.css']
+  styleUrls: ['./team.component.css'],
 })
 export class TeamComponent {
+  constructor(
+    private teamService: TeamService,
+    private route: ActivatedRoute
+  ) {}
 
+  teamName: string | null = 'error';
```

Comment:
```suggestion
  teamId: string | null = 'error';
```

Reply:
Refactored that a bit to make it more consistent with the club component